tests/suites/test_suite_platform_printf.function

```diff
@@ -31,6 +31,31 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void printf_long_max(const char *format, long value)
```

Comment:
See https://github.com/Mbed-TLS/mbedtls/pull/6716#discussion_r1178219165 . We'd need `long long` (mandatory in C99, but it would mean requiring a `printf` implementation that supports it, and that's a higher bar), or a test case that's conditional on `LONG_MAX > …`.